__tests__/setup/elements/getScrollToBottomButton.js

```diff
@@ -5,5 +5,5 @@ import getTranscript from './getTranscript';
 export default async function getScrollToBottomButton(driver) {
   const transcript = await getTranscript(driver);
 
-  return await transcript.findElement(By.css('.webchat__scrollToEndButton'));
+  return await transcript.findElement(By.css('.webchat__scroll-to-end-button'));
```

Comment:
This is ok for BEM naming convention?

Reply:
Yap, this is BEM naming and this is B-only.